src/OrchardCore.Modules/OrchardCore.Workflows/Trimming/Services/WorkflowTrimmingBackgroundTask.cs

```diff
@@ -0,0 +1,62 @@
+using System;
```

Comment:
so strange OC is not using implicit usings yet :(

At least file scoped namespaces made it in.

Reply:
On it's way https://github.com/OrchardCMS/OrchardCore/pull/16574